src/realtime_metrics/main.py

```diff
@@ -148,6 +180,52 @@ def eta_accuracy(stop_time_updates: list[tuple[TripUpdate, StopTimeUpdate]]):
     return mean_accuracy
 
 
+def availability_acceptable_stop_time_updates(stop_time_updates: list[tuple[TripUpdate, StopTimeUpdate]], 
+                                              time_frame_start: int, 
+                                              time_frame_end: int):
+    """
+    computes the availability of acceptable stop time updates metrics for the given stop time updates in the given time frame.
+    stop_time_updates: list of corresponding trip updates and stop time updates
+    time_frame_start: start of the time frame to observe, in minutes since 1970
+    time_frame_end: end of the time frame to observe, in minutes since 1970
+    """
+    logger.info("Time frame start: %s", time_frame_start)
+    logger.info("Time frame end: %s", time_frame_end)
+
+    # dict to store how many stop time updates are available for each minute slot
+    time_slots_dict: Dict[int, int] = dict()
+
+    for update in stop_time_updates:
+        trip_update = update[0]
+
+        # get time in minutes
+        time_in_minutes = int(trip_update.timestamp.timestamp() / 60)
+
+        # skip, if outide of time frame
+        if time_in_minutes < time_frame_start or time_in_minutes > time_frame_end:
+            continue
+
+        # increase respective stop time counter
+        if time_in_minutes not in time_slots_dict.keys():
+            time_slots_dict[time_in_minutes] = 1
+        else:
+            time_slots_dict[time_in_minutes] = time_slots_dict[time_in_minutes] + 1
+
+    # calculate percentage of minutes with two or more stop time updates
+    time_slots = time_frame_end - time_frame_start + 1
+    logger.info("Time slots: %s", time_slots)
+
+    time_slots_with_two_entries = 0
+    for key in time_slots_dict.keys():
+        if time_slots_dict[key] >= 2:
+            time_slots_with_two_entries += 1
+    logger.info("Time slots with at least two updates: %s", time_slots_with_two_entries)
+
+    percentage = (time_slots_with_two_entries / time_slots) * 100
```

Comment:
Percentage is rather the unit, so I would name it as what it is or remove the variable altogether.
```suggestion
    availability_acceptable_stop_time_updates = (time_slots_with_two_entries / time_slots) * 100
```